package.json

```diff
@@ -14,16 +14,16 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^4.0.0",
-    "@angular/common": "^4.0.0",
-    "@angular/compiler": "^4.0.0",
-    "@angular/core": "^4.0.0",
-    "@angular/forms": "^4.0.0",
-    "@angular/http": "^4.0.0",
-    "@angular/platform-browser": "^4.0.0",
-    "@angular/platform-browser-dynamic": "^4.0.0",
-    "@angular/router": "^4.0.0",
-    "@ngrx/store": "^4.0.3",
+    "@angular/animations": "^4.4.6",
```

Comment:
higher numbers are more better, right? I saw that adflow had a more recent angular version, so I added @latest here. Seemed safe enough since we're in active development.

Reply:
what could go wrong? 😸